libclamav/yara_compiler.c

```diff
@@ -420,64 +420,73 @@ int _yr_compiler_compile_rules(
 
     if (result == ERROR_SUCCESS) {
         compiler->automaton_arena = NULL;
-        result                    = yr_arena_append(
-                               arena,
-                               compiler->code_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->code_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->code_arena = NULL;
-        result               = yr_arena_append(
-                          arena,
-                          compiler->re_code_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->re_code_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->re_code_arena = NULL;
-        result                  = yr_arena_append(
-                             arena,
-                             compiler->rules_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->rules_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->rules_arena = NULL;
-        result                = yr_arena_append(
-                           arena,
-                           compiler->strings_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->strings_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->strings_arena = NULL;
-        result                  = yr_arena_append(
-                             arena,
-                             compiler->externals_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->externals_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->externals_arena = NULL;
-        result                    = yr_arena_append(
-                               arena,
-                               compiler->namespaces_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->namespaces_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->namespaces_arena = NULL;
-        result                     = yr_arena_append(
-                                arena,
-                                compiler->metas_arena);
+
+        result = yr_arena_append(
+            arena,
+            compiler->metas_arena);
     }
 
     if (result == ERROR_SUCCESS) {
         compiler->metas_arena = NULL;
-        result                = yr_arena_append(
-                           arena,
-                           compiler->sz_arena);
+
+        result = yr_arena_append(
+            arena,
```

Comment:
Per our offline conversation, the rest of `yara_compiler.c` already follows the convention of having all arguments on a separate line from the function call. It's fairly common elsewhere in the project. 

I also think it would look better if we could enforce putting the first argument on the same line, which would align them, but `clang-format` doesn't appear to have an option to do this automatically.  The closest thing I could find is `AllowAllArgumentsOnNextLine false`, which doesn't help. That one is for when the line is really long so it is allows to move the params to a new line.  

Anyways, per our convo, it's not worth the effort to manually refactor them all, so we'll just live with it.